app/controllers/api/v0/competitions_controller.rb

```diff
@@ -4,6 +4,14 @@ class Api::V0::CompetitionsController < Api::V0::ApiController
   # Enable CSRF protection if we use cookies based user instead of OAuth one.
   protect_from_forgery if: -> { current_user.present? }, with: :exception
 
+  def wrc
+    competition = competition_from_params
+    delegate_report = competition.delegate_report
+
+
+    render json: wrc_information.to_json
```

Comment:
What is `wrc_information` and where does it come from?

Reply:
This was just from playing around with payload generation